### ISSUE TITLE
Image build fails on tale using older r2d

### ISSUE THREAD

**Issue by craig-willis** [whole-tale/whole-tale#125]
I ran into this while testing https://github.com/whole-tale/ngx-dashboard/pull/281. We updated gwvolman to pass `--engine dockercli` to r2d, but this fails for tales published using older r2d.

**Steps to Reproduce**

* Go to https://sandbox.zenodo.org/record/1059441 and select "Run this Tale on Whole Tale"
* After import, select "Run Tale"

**Expected Results**

Build succeeds using older version of r2d

**Actual Results**

Build fails because of unsupported flag:

```Forcing build.
repo ...
jupyter-repo2docker: error: unrecognized arguments: --engine
ValueError: Error building tale 62c7f66cd4f8c34e134a378b
  File "/usr/local/lib/python3.8/dist-packages/celery/app/trace.py", line 412, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/girder_worker/task.py", line 148, in __call__
    results = super(Task, self).__call__(*_t_args, **_t_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/celery/app/trace.py", line 704, in __protected_call__
    return self.run(*args, **kwargs)
  File "/gwvolman/gwvolman/tasks.py", line 363, in build_tale_image
    raise ValueError('Error building tale {}'.format(tale_id))
```

**Notes**
There is no convenient way for the user to build using the newer r2d.

**Comment by Xarthisius** [comment]
That's a very good catch. However, technically we haven't done it yet :) It was a part of testing steps for https://github.com/whole-tale/repo2docker_wholetale/pull/44 I'll make sure to keep this in mind when I'll issue a PR for this.

**Comment by craig-willis** [comment]
Fixed and verified in v1.2rc2
